test/models/merged_search_service_test.rb

```diff
@@ -0,0 +1,202 @@
+require 'test_helper'
+require 'ostruct'
```

Comment:
Are we only using OpenStruct in testing or has it ended up on the running app as well?

Zero concerns if it's just in tests, minor nervousness if it is in the main codebase because it has the _potential_ to cause unexpected slow downs. Mostly we'd just want to keep an eye on it, not avoid its use.

Reply:
We're just using it in testing.